diff --git a/base/BUILD.gn b/base/BUILD.gn
index 3b2a1d31aeab17c1153e7ae351d8cfa79ec3bf43..d9ccb93acbeaf88ea235a4d60bca23b39ada28e0 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4479,6 +4479,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4514,6 +4515,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_aidl("process_launcher_aidl") {
