diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index c5ffcf0672d78aba5181dd36063ded72ce4c9534..e3f51961bf23acb611109ff9b27266ea2e1fb48e 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -344,6 +344,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
