diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index dee39a8c59c25de109c248ae90a5ae176f5d1f5c..cb6d11d6db4b3cb14b0b9c62e9f63841a0ab1326 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2720,6 +2720,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
