diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
index 7d6b76dda72ae4ee7b4e374ea09fe72c9a0b7df3..484210b9c915df534c7c5d3d5069bc6c2646044f 100644
--- a/chrome/browser/ui/browser_navigator.cc
+++ b/chrome/browser/ui/browser_navigator.cc
@@ -586,6 +586,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
     // Block any navigation requests in locked fullscreen mode.
     return nullptr;
   }
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/1096345): Remove this code after we integrate with intent
@@ -932,6 +933,7 @@ bool IsHostAllowedInIncognito(const GURL& url) {
   base::StringPiece host = url.host_piece();
   if (scheme != content::kChromeUIScheme)
     return true;
+  BRAVE_IS_HOST_ALLOWED_IN_INCOGNITO
 
   if (host == chrome::kChromeUIChromeSigninHost) {
 #if BUILDFLAG(IS_WIN)
