diff --git a/chrome/browser/ui/views/bubble/webui_bubble_manager.h b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
index c62977d27e45c72fd0f8d661ed90d08914baaae4..b23aee971b4b7893a8a98e54cd23d2fb06c087a3 100644
--- a/chrome/browser/ui/views/bubble/webui_bubble_manager.h
+++ b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
@@ -222,6 +222,7 @@ WebUIBubbleManagerImpl<T>::CreateWebUIBubbleDialog(
 
   auto bubble_view = std::make_unique<WebUIBubbleDialogView>(
       anchor_view_, contents_wrapper->GetWeakPtr(), anchor, arrow);
+    BRAVE_WEBUI_BUBBLE_MANAGER_T_CREATE_WEB_UI_BUBBLE_DIALOG
 
   // Register callback to emit histogram when the widget is created
   if (bubble_init_start_time_) {
