diff --git a/chrome/browser/android/browsing_data/browsing_data_bridge.cc b/chrome/browser/android/browsing_data/browsing_data_bridge.cc
index d738954e50f971ba7bd22c8147c9d31fb75a8788..5a956713b9efb641b0b46342c4c7c1ac2e487f80 100644
--- a/chrome/browser/android/browsing_data/browsing_data_bridge.cc
+++ b/chrome/browser/android/browsing_data/browsing_data_bridge.cc
@@ -120,6 +120,7 @@ static void JNI_BrowsingDataBridge_ClearBrowsingData(
         remove_mask |= chrome_browsing_data_remover::DATA_TYPE_CONTENT_SETTINGS;
         break;
       case browsing_data::BrowsingDataType::DOWNLOADS:
+      BRAVE_CLEAR_BROWSING_DATA
       case browsing_data::BrowsingDataType::HOSTED_APPS_DATA:
         // Only implemented on Desktop.
         NOTREACHED();
