diff --git a/chrome/browser/ui/omnibox/chrome_omnibox_client.h b/chrome/browser/ui/omnibox/chrome_omnibox_client.h
index 498d21fd3d562b6e5b370b91d6d57ac01bc7639f..ba2198815da73977210f8ff73a24d78326a4e8d5 100644
--- a/chrome/browser/ui/omnibox/chrome_omnibox_client.h
+++ b/chrome/browser/ui/omnibox/chrome_omnibox_client.h
@@ -23,7 +23,7 @@ class GURL;
 class LocationBar;
 class Profile;
 
-class ChromeOmniboxClient final : public OmniboxClient {
+class ChromeOmniboxClient : public OmniboxClient {
  public:
   ChromeOmniboxClient(LocationBar* location_bar,
                       Browser* browser,
