diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index 61101b6cecf5a7f5aba49c331ae935222d50148e..a8e8a0aa7ab2ea4f0ecebe66b5d59f7fe91bcc96 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -233,6 +233,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
