diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 57a81489dae0e6bdc2e15fec1f38329b96341edf..ae4a5c3c7eff7b2699d05a28aa84a39009591866 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2695,6 +2695,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
