diff --git a/chrome/browser/ui/views/bubble/webui_bubble_manager.h b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
index a554528ce9c2795bd70c4d2fadf887657a3418bf..7282dc3e11c370ccdc71fddd4f4e0269a1c62a0b 100644
--- a/chrome/browser/ui/views/bubble/webui_bubble_manager.h
+++ b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
@@ -165,9 +165,11 @@ class WebUIBubbleManagerImpl : public WebUIBubbleManager {
  private:
   // WebUIBubbleManager:
   void MaybeInitPersistentRenderer() override;
+ protected:
   base::WeakPtr<WebUIBubbleDialogView> CreateWebUIBubbleDialog(
       const std::optional<gfx::Rect>& anchor,
       views::BubbleBorder::Arrow arrow) override;
+ private:
   WebUIContentsWrapper* GetContentsWrapper() override;
 
   const raw_ptr<views::View> anchor_view_;
@@ -247,6 +249,7 @@ WebUIBubbleManagerImpl<T>::CreateWebUIBubbleDialog(
 
   auto bubble_view = std::make_unique<WebUIBubbleDialogView>(
       anchor_view_, contents_wrapper->GetWeakPtr(), anchor, arrow);
+    BRAVE_WEBUI_BUBBLE_MANAGER_T_CREATE_WEB_UI_BUBBLE_DIALOG
 
   if (!widget_initialization_callback_.is_null()) {
     bubble_view->RegisterWidgetInitializedCallback(
