diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index d8fd6c19e69c8d3da736dc5e2f0949db91094ae8..f04419d6952601943b7444d2b14154741490b6d1 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3942,6 +3942,7 @@ const FeatureEntry::FeatureVariation kAndroidHubVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
