diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 0861c2ac8982a5d130c3e968443c6cbe828380f3..77aac9595f631d00c8c6108660c5531198a72523 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -77,4 +77,5 @@ source_set("configs") {
       deps += [ "//ash/webui/sample_system_web_app_ui" ]
     }
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
