diff --git a/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc b/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc
index c4944cd24ff8314424691f818979f405f55bf7ae..05fe2098161dcec83b74ef7ec580be2c6afaa333 100644
--- a/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc
+++ b/chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc
@@ -909,6 +909,7 @@ void ChromeWebUIControllerFactory::GetFaviconForURL(
 
 // static
 ChromeWebUIControllerFactory* ChromeWebUIControllerFactory::GetInstance() {
+  BRAVE_CHROME_WEBUI_CONTROLLER_FACTORY
   static base::NoDestructor<ChromeWebUIControllerFactory> instance;
   return instance.get();
 }
