diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index d3853eba47647c01f112adfb4fb7ade2d35e5ec9..ffe1bfeadde5b2be1e6695cbfb5a52de81fb2e6a 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2748,6 +2748,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
