diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 74e37236658a88becb5a4c5065d88fd3f11d22bb..d5adc6d3704a2f9121363745c991c464d4b7c8f4 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3975,6 +3975,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
