diff --git a/chrome/browser/ui/views/bubble/webui_bubble_manager.h b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
index 875aecf6a900872b47ec858536b7d668a5778e2a..34c14b48fb26222f5557fd30d583b633fe065c01 100644
--- a/chrome/browser/ui/views/bubble/webui_bubble_manager.h
+++ b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
@@ -169,9 +169,11 @@ class WebUIBubbleManagerImpl : public WebUIBubbleManager {
  private:
   // WebUIBubbleManager:
   void MaybeInitPersistentRenderer() override;
+ protected:
   base::WeakPtr<WebUIBubbleDialogView> CreateWebUIBubbleDialog(
       const std::optional<gfx::Rect>& anchor,
       views::BubbleBorder::Arrow arrow) override;
+ private:
   WebUIContentsWrapper* GetContentsWrapper() override;
 
   const raw_ptr<views::View> anchor_view_;
@@ -266,6 +268,7 @@ WebUIBubbleManagerImpl<T>::CreateWebUIBubbleDialog(
 
   auto bubble_view = std::make_unique<WebUIBubbleDialogView>(
       anchor_view_, contents_wrapper->GetWeakPtr(), anchor, arrow);
+    BRAVE_WEBUI_BUBBLE_MANAGER_T_CREATE_WEB_UI_BUBBLE_DIALOG
 
   if (!widget_initialization_callback_.is_null()) {
     bubble_view->RegisterWidgetInitializedCallback(
