diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 57721d673d22bdda54eaa3e31884a93658acb424..fcf031e58536c04a356727c6ed9dadde1fcede8e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3735,6 +3735,7 @@ const FeatureEntry::FeatureVariation kLinkPreviewTriggerTypeVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
@@ -11183,6 +11184,7 @@ void GetStorage(Profile* profile, GetStorageCallback callback) {
 
 bool ShouldSkipConditionalFeatureEntry(const flags_ui::FlagsStorage* storage,
                                        const FeatureEntry& entry) {
+  BRAVE_SHOULD_SKIP_CONDITIONAL_FEATURE_ENTRY
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   version_info::Channel channel = chrome::GetChannel();
   // enable-projector-server-side-speech-recognition is only available if
