diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 2b1f33f52d2e0acf3f6ba9c7994c7ee73c685f1f..f9c01aed5ffee024c26d5c6463b77c7373182599 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -142,6 +142,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kIdentityProvider:
       return RequestTypeForUma::PERMISSION_IDENTITY_PROVIDER;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
