diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 6084f14eabf71e745467e59106a318d84f076453..71b29c44fb0ce17f2f5bfe13c02ab2f126f9716b 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1248,6 +1248,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
