diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 16fc52f006912ea333ae81ca7d14579418ee8bd9..8c269d9f344300f86c1f92829337dd5bbaf99f4b 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -135,6 +135,7 @@ android_library("java") {
     "//ui/android:ui_java",
   ]
 
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
