diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 2f3f2de49a5b1d4ae99e8af7f1a5eb315d1af63b..b1a6cb459a51d075fd5c120af566f8cebe9d7925 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -266,6 +266,7 @@ class SyncServiceImpl : public SyncService,
   void ReportDataTypeErrorForTest(ModelType type);
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
