diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 57ccbe1054ee78bd3e5bcd2c741c24ced3d86bf2..939fa8f1a1431868e70c7e03e396fa425f3590c7 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -252,6 +252,7 @@ class SyncServiceImpl : public SyncService,
   bool IsSyncFeatureConsideredRequested() const override;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
