diff --git a/components/content_settings/core/common/content_settings_types.mojom b/components/content_settings/core/common/content_settings_types.mojom
index e62e947cc5715b64fc178f3f4ed9fbb4779f58f5..1dc2cd19cf575f5b4811e523e9aa14fb1afbb6c3 100644
--- a/components/content_settings/core/common/content_settings_types.mojom
+++ b/components/content_settings/core/common/content_settings_types.mojom
@@ -409,5 +409,21 @@ enum ContentSettingsType {
   // to mouse inputs.
   POINTER_LOCK,
 
+  BRAVE_START,
+  BRAVE_ADS = BRAVE_START,
+  BRAVE_COSMETIC_FILTERING,
+  BRAVE_TRACKERS,
+  BRAVE_HTTP_UPGRADABLE_RESOURCES,
+  BRAVE_FINGERPRINTING_V2,
+  BRAVE_SHIELDS,
+  BRAVE_REFERRERS,
+  BRAVE_COOKIES,
+  BRAVE_SPEEDREADER,
+  BRAVE_ETHEREUM,
+  BRAVE_SOLANA,
+  BRAVE_GOOGLE_SIGN_IN,
+  BRAVE_HTTPS_UPGRADE,
+  BRAVE_REMEMBER_1P_STORAGE,
+  BRAVE_LOCALHOST_ACCESS,
   NUM_TYPES,
 };
