diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 42452c1b02eb72a2d9f5f22f25e3837f5121f964..a9279de05d1b9a6d20897ad7b75019b715e2a1db 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -536,7 +536,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
   if (!IsAllowed(setting)) {
     return BlockAllCookies{};
   }
-
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!ShouldBlockThirdPartyCookies() &&
       !Are3pcsForceDisabledByOverride(overrides)) {
     return AllowAllCookies{
@@ -641,6 +641,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   const ContentSetting cookie_setting = GetContentSetting(
       url, first_party_url, ContentSettingsType::COOKIES, &setting_info);
 
+  BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL
   const bool is_explicit_setting =
       !setting_info.primary_pattern.MatchesAllHosts() ||
       !setting_info.secondary_pattern.MatchesAllHosts();
