diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 1952b5018f8e5bed87b7dc9d8012c8f2fac00c96..66414ae627da9e4e38ed147d92b062660da1159e 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -883,6 +883,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
