diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 77dfb484c02959d423afdc9b7c5b469982806b7f..2b43239fa275532eda8d19c6e3bc756c2db78ce0 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -884,6 +884,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
