diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index f46edeebd1c5e3350dea2f0549ed2a24cfad7a75..b568c5e49a4ec7938853df9416801e3bf08e3d45 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -883,6 +883,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
