diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index b811241aee342a07aecbeab1ea342e0829cf34d9..dea0f3ce722ff726969cb9a1301b43b20feebcb5 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -189,6 +189,7 @@ component("cpp") {
     "//services/proxy_resolver/public/mojom",
   ]
 
+  public_deps += [ "//brave/services/network/public/cpp" ]
   defines = [ "IS_NETWORK_CPP_IMPL" ]
 }
 
