diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index f7145fcfe9a4d531521db202285bca572d600d8c..2d584f022c14969a9edfbb09907154a575bf9cf3 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -465,6 +465,7 @@ java_prebuilt("com_google_auto_value_auto_value_annotations_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
