diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index 6ac69efdd662b72d468f641d163b365186f5db69..d46197e6c5cfa42744d20d9154915b5bc8909663 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -1735,6 +1735,7 @@
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
       status: "experimental",
+      base_feature: "none",
     },
     {
       // The FileSystemHandle.getCloudIdentifiers() method (see
@@ -1750,6 +1751,7 @@
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
       status: {"Android": "test", "default": "stable"},
+      base_feature: "none",
     },
     {
       name: "FileSystemAccessLockingScheme",
@@ -2467,6 +2469,7 @@
     {
       name: "MiddleClickAutoscroll",
       status: "test",
+      base_feature: "none",
     },
     {
       name: "MobileLayoutTheme",
@@ -3464,6 +3467,7 @@
     {
       name: "Serial",
       status: {"Android": "", "default": "stable"},
+      base_feature: "none",
     },
     {
       name: "SerializeViewTransitionStateInSPA",
