diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index 40231b8d67a6e7a72240141dc12e0c769744aa6c..964830b82cd1a1aa177f61d291f04129cdc1a00d 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -465,6 +465,7 @@ java_prebuilt("com_google_auto_value_auto_value_annotations_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
