diff --git a/BUILD.gn b/BUILD.gn
index 186c652a0e55e7b20012ec929fd645a921cd9729..f0d5d12949834a6602411b27cefd6814e2724028 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -747,6 +747,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3113,6 +3114,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
