diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 38b7faa74ab5a380ad42582753856eadc2e86b1a..ef3aafede301ab6c2a5bab1640ad6348fea1771a 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2989,6 +2989,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
