diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 314bd42aceb500ff2e441bda3efa0ef8c1f378eb..716e48726ad5d6ec6c42d8a166a5a60b90d12baa 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3013,6 +3013,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
