diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 1a9af3c8bfbf782ebfc12becf0ecf5ec37e1eccd..22620bc54d77b573adb578ab10aa4304fe5b5795 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3015,6 +3015,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
